cmd/cli/metrics_test.go

```diff
@@ -124,7 +131,9 @@ func TestRun_MetricsEnable(t *testing.T) {
 		}
 
 		err := scenario.cmd.run()
-		assert.Nil(err)
+		if err != nil {
+			assert.EqualError(err, "no namespace mentioned")
```

Comment:
I have a prefrence to augment the testSecenario to include something like `errExpected: errstring`.  This allows for the test to be more explict and allow for other test to have different errors as well.  for instance if a name space didn't exist we could set an expected error

Reply:
Thanks, @jsturtevant for the review